src/registry.rs

```diff
@@ -290,10 +290,10 @@ impl SystemRegistry {
     }
 
     /// Add new actor to the registry by address, panic if actor is already running
-    pub fn set<A: SystemService + Actor<Context = Context<A>>>(&self, addr: Addr<A>) {
+    pub fn set<A: SystemService + Actor<Context = Context<A>>>(&self, addr: &Addr<A>) {
```

Comment:
hm... This is technically breaking change as it is public interface.

I think instead we should not clone addr below at line 302.
The effect would be the same

Reply:
:+1:  :)